gstd/src/msg/basic.rs

```diff
@@ -215,19 +218,21 @@ pub fn reply_bytes<T: AsRef<[u8]>>(payload: T, value: u128) -> MessageId {
 ///
 /// pub unsafe extern "C" fn handle() {
 ///     // ...
-///     msg::reply_push(b"Part 1");
+///     msg::reply_push(b"Part 1").unwrap();
 ///     // ...
-///     msg::reply_push(b"Part 2");
+///     msg::reply_push(b"Part 2").unwrap();
 ///     // ...
-///     msg::reply_commit(42);
+///     msg::reply_commit(42).unwrap();
 /// }
 /// ```
 ///
 /// # See also
 ///
 /// [`reply_push`] function allows to form a reply message in parts.
-pub fn reply_commit(value: u128) -> MessageId {
-    gcore::msg::reply_commit(value).into()
+pub fn reply_commit(value: u128) -> Result<MessageId> {
```

Comment:
And how do you feel about adding `#[must_use]` for end-sending functions (reply_commit, reply, send, send_commit) ?

Reply:
`#[muse_use]` is already applied to `Result` so it's redundant